src/main/resources/db/migration/V7__Add_utbet_tom.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE UTBETALING_SPLEIS ADD COLUMN UTBET_TOM DATE;
+
+UPDATE UTBETALING_SPLEIS SET UTBET_TOM = TOM;
```

Comment:
Defaultverdien for gamle rader blir den verdien som har blitt brukt fram til nå.